plugins/extractit/icon_extractit/util/extractor.py

```diff
@@ -15,12 +15,23 @@
 def extract(provided_regex: str, provided_string: str, provided_file: str) -> list:
     matches = []
     if provided_string:
-        provided_string = urllib.parse.unquote(provided_string)
-        matches = regex.findall(provided_regex, provided_string)
+        mixedmatches = regex.findall(provided_regex, provided_string)  # regex finds both encoded and unencoded urls
+        for match in mixedmatches:
+            decoded = urllib.parse.unquote(match)
+            if decoded != match:
+                matches.append(match)
+            else:
+                matches.append(decoded)
```

Comment:
There doesn't seem to be any negative effect to unquoting/decoding a URL twice, so I think you can remove the if/else and just append `decoded` every time. Same applies to lines 31 - 34.